projects/plugins/jetpack/extensions/blocks/videopress/seekbar-color-settings.js

```diff
@@ -43,33 +43,50 @@ class SeekbarColorSettings extends Component {
 
 	render() {
 		const { seekbarColor, seekbarPlayedColor, seekbarLoadingColor } = this.state;
+		const { toggleAttribute, useAverageColor } = this.props;
+		const showTitle = true;
 
 		return (
-			<PanelColorSettings
-				title={ __( 'Progress Colors', 'jetpack' ) }
+			<PanelBody
+				className="seekbar-color-settings__panel"
+				title={ __( 'Progress Bar Colors', 'jetpack' ) }
 				initialOpen={ false }
-				colorSettings={ [
-					{
-						value: seekbarColor,
-						onChange: this.handleChangeSeekbarColor,
-						label: __( 'Main', 'jetpack' ),
-					},
-					{
-						value: seekbarLoadingColor,
-						onChange: this.handleChangeSeekbarLoadingColor,
-						label: __( 'Loaded', 'jetpack' ),
-					},
-					{
-						value: seekbarPlayedColor,
-						onChange: this.handleChangeSeekbarPlayedColor,
-						label: __( 'Progress', 'jetpack' ),
-					},
-				] }
 			>
-				<Button isDefault onClick={ this.saveColors }>
-					{ __( 'Save colors', 'jetpack' ) }
-				</Button>
-			</PanelColorSettings>
+				<ToggleControl
+					label={ __( 'Match video', 'jetpack' ) }
+					help={ __( 'Colors adapt to the video as it plays', 'jetpack' ) }
+					onChange={ toggleAttribute( 'useAverageColor' ) }
+					checked={ useAverageColor }
+				/>
+				<PanelColorSettings
+					opened={ ! useAverageColor }
+					showTitle={ false }
+					colorSettings={ [
+						{
+							value: seekbarColor,
+							onChange: this.handleChangeSeekbarColor,
+							label: __( 'Main', 'jetpack' ),
+							showTitle
+						},
+						{
+							value: seekbarLoadingColor,
+							onChange: this.handleChangeSeekbarLoadingColor,
+							label: __( 'Loaded', 'jetpack' ),
+							showTitle
+						},
+						{
+							value: seekbarPlayedColor,
+							onChange: this.handleChangeSeekbarPlayedColor,
+							label: __( 'Progress', 'jetpack' ),
+							showTitle
+						},
+					] }
+				>
+					<Button isDefault onClick={ this.saveColors }>
```

Comment:
Can we take that opportunity to update the button here and get rid of the deprecation notice?

```
Button isDefault prop is deprecated since version 5.4. Please use variant="secondary" instead. 
```